lib/gooddata/helpers/splunk_helper.rb

```diff
@@ -0,0 +1,55 @@
+require 'rest-client'
+require 'nokogiri'
+
+require_relative 'global_helpers'
+
+module GoodData
+  attr_accessor :splunk_api_helper
+
+  def splunk_api_helper
+    if @splunk_api_helper.nil?
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.